clients/cobol-lsp-vscode-extension/src/__tests__/services/copybook/CopybooksCodeActionProviderTest.spec.ts

```diff
@@ -12,10 +12,14 @@
  *   Broadcom, Inc. - initial API and implementation
  */
 
+jest.mock("../../../services/reporter", () => ({
+  registerEvent: jest.fn(),
+}));
```

Comment:
why isn't the following enough like in the other modules?
```suggestion
jest.mock("../../../services/reporter");
```